components/rhs_comment/rhs_comment.jsx

```diff
@@ -143,7 +143,6 @@ export default class RhsComment extends React.Component {
             <PostTime
                 isPermalink={isPermalink}
                 eventTime={post.create_at}
-                useMilitaryTime={this.props.useMilitaryTime}
```

Comment:
For these, can you remove whatever is passing them `useMilitaryTime` if they aren't using it any more?